src/modules/account/rest-api/account-controller.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from 'express';
+import AccountService from '../account-service';
+import { Account, AccountSearchParams, CreateAccountParams } from '../types';
+
+export default class AccountController {
+  public static async createAccount(
+    req: Request,
+    res: Response,
+  ): Promise<void> {
+    const { username, password }: CreateAccountParams = req.body as CreateAccountParams;
+    const params: CreateAccountParams = { username, password };
+    const account = await AccountService.createAccount(params);
+    res.status(201).send(AccountController.serializeAccountAsJSON(account));
+  }
+
+  public static async getAccountByUsername(
```

Comment:
Let's remove this. It may be needed on specific application to find user by username but generally, not